test/testdata/deployednettemplates/recipes/README.md

```diff
@@ -0,0 +1,45 @@
+# Recipes
+
+Most of the recipes' net.json and genesis.json use one of the following methods to call `netgoal generate`:
+1. `Makefile`
+2. `python3 {GO_ALGORAND_PATH}/test/testdata/deployednettemplates/generate-recipe/generate_network.py -f {PATH_TO}/network-tpl.json`
+
+Details for netgoal generate could be found in the binary with:
+```
+netgoal generate -h
+```
+
+Source code for netgoal can be found in `{GO_ALGORAND_PATH}/cmd/netgoal/generate.go`
+
+Quick reference when reading some of the Makefiles in this directory:
+```
+PARAMS=-w 100 -R 8 -N 20 -n 100 -H 10 -X 10 --node-template node.json --relay-template relay.json --non-participating-node-template nonPartNode.json
```

Comment:
should we make this example use full named flags instead of shorthand, so people don't have to translate what it means?

Reply:
w, N vs n, H, X are not intuitive enough to know what they mean without looking. We can probably get rid of them to avoid mistakes

Reply:
The Makefiles inside the scenarios use short-hand, so this is for a quicker reference to what the short-hand flags mean. I thought about updating all of the scenarios to long-form ... but I wasn't too comfortable doing it all by hand and manually testing since I feel like I can easily mess that up by accident.

Instead, maybe for new scenarios we can use long-form if we're calling netgoal directly using a Makefile, we can use long-form. The long-form flags are all provided in the README. Plus, if they use the python way of generating the scenario, then they don't have to worry about any of the flags.

Reply:
Let me think about updating it. Maybe I can create a spreadsheet and go through the scenarios one by one and test. It will take some time.

Reply:
i just meant in the example at least. So we set a good example for people to use.

Reply:
plus you are adding `-X` to some files, we should maybe add it's full name.

Reply:
there's a thread on slack. The current full names are also pretty ambiguous. I'm working on updating the flags and considering updating the variable names in the cmd/netgoal/generate.go file

Reply:
renamed the long-form names and added examples using long-form and modified references to the new flag "-X" with the new long-form flag